pkg/checks/external/nodeCheck/main.go

```diff
@@ -0,0 +1,203 @@
+package nodeCheck
+
+import (
+	"errors"
+	"net"
+	"os"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/pkg/api/v1/pod"
+)
+
+// WaitForKuberhealthyEndpoint first checks the node's age to make sure it's not less than three minutes old. If so, it
+// checks if the kuberhealthy endpoint (KH_REPORTING_URL) can be hit, determining whether or not the proper netfilter
+// rules have been set up on the node in order to hit the kuberhealthy service.
+func WaitForKuberhealthyEndpoint(client *kubernetes.Clientset, namespace string, kuberhealthyEndpoint string) {
```

Comment:
Perhaps we don't wait 3 minutes here... simply scope this down to just checking the KH endpoint?  If they want to wait 3 minutes, or wait for the kube-proxy pod to come up, they can use the other funcs added here.